.github/workflows/example.yml

```diff
@@ -0,0 +1,34 @@
+name: PROS Build Example
+
+on:
+    push:
+      branches: "**"
+    pull_request:
+      branches: "**"
+  
+    workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run abucky0/pros-build
+        id: test
+        uses: LemLib/pros-build
+        with:
+          multithreading: true
+          no_commit_hash: false
+
+      - name: Upload Artifact
```

Comment:
Why not handle uploading the artifact inside of `pros-build`?

Reply:
Because I couldn’t figure out how to do it from the script. It’s possible that I could do it using the GitHub app but idk.

Reply:
Oh and it also allows more control over the process, such as instances where you might not want to upload an artifact, such as only uploading artifacts on pull request.